src/wp-includes/block-supports/block-style-variations.php

```diff
@@ -224,14 +224,13 @@ function wp_render_block_style_variation_class_name( $block_content, $block ) {
  *
  * @return array Block variations data to be merged under `styles.blocks`.
  */
-function wp_resolve_and_register_block_style_variations( $variations ) {
+function wp_resolve_block_style_variations( $variations ) {
```

Comment:
Removed the `_register` from the name as it no longer registers them.

Reply:
We'll need to update the docblock description as well.

Reply:
Fixed it.